installers/linux/archlinux/PKGBUILD

```diff
@@ -1,8 +1,11 @@
 # vim: ts=2 sts=2 sw=2 et ft=sh
```

Comment:
Should we git ignore these files since they're not auto-generated, or do we still need to check them in?